src/_data/styles/spectrum.json

```
@@ -0,0 +1,7 @@
+{
+	"title": "Spectrum",
+	"author": "Robert Manders",
+	"stylesheet": "https://github.com/Rob-Manders/Style-Stage-Spectrum/blob/main/spectrum.css",
```

Little update to use the "raw" file. 👍
```suggestion
	"stylesheet": "https://raw.githubusercontent.com/Rob-Manders/Style-Stage-Spectrum/main/spectrum.css",
```